src/main/java/com/amalitech/gpuconfigurator/controller/AuthenticationResponse.java

```diff
@@ -0,0 +1,18 @@
+package com.amalitech.gpuconfigurator.controller;
+
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+
+@Data
+@Builder
+@AllArgsConstructor
+@NoArgsConstructor
+public class AuthenticationResponse {
```

Comment:
please can we use records instead of pojos for all our dtos. it is much more cleaner